plugins/woocommerce/src/Internal/DataStores/Orders/FeatureController.php

```diff
@@ -271,7 +271,7 @@ private function delete_custom_orders_table() {
 	 * @return array The updated settings sections array.
 	 */
 	private function get_settings_sections( array $sections ): array {
-		if ( ! $this->is_feature_visible() || ! $this->data_synchronizer->check_orders_table_exists() ) {
```

Comment:
The expected user flow at this early stage of development is:

1. User adds the code snippet to enable the feature.
2. User uses the dedicated tool to create the table.
3. User can then go and see and modify the related settings.

So, 3 doesn't make much sense until 1 and 2 have been done. Given that, I think that the `check_orders_table_exists` check should remain for now (for sure we'll remove it in the future, but not yet).

Reply:
As discussed, I have changed a bit to still show the setting if feature is enabled, but also added a message to go to the tools to generate tables when they don't exists in https://github.com/woocommerce/woocommerce/pull/31811/commits/fc0c45a204aad1a2d629351ca40acca11187e712.